recipes/qtxlsxwriter/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+import os
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain
+from conans import ConanFile, tools
+
+
+class QtXlsxWriterConan(ConanFile):
+    name = "qtxlsxwriter"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/dbzhang800/QtXlsxWriter"
+    description = ".xlsx file reader and writer for Qt5"
+    topics = ("qtxlsxwriter", "excel", "xlsx", "conan-recipe")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
+
+    generators = "CMakeDeps"
+    exports_sources = [
+        "CMakeLists.txt",
+        os.path.join("patches", "*")
+    ]
```

Comment:
This is internal logic of Conan so we can keep it short

```suggestion
    exports_sources = "CMakeLists.txt", "patches/**"
```

Reply:
[Done](https://github.com/conan-io/conan-center-index/pull/4838/commits/ad5922d7d45e0579a2fc70b0d670cc62fec9c008)